### PR TITLE
DX-2200 Voice, Answering machine detection

### PR DESCRIPTION
**Pull request by hamermike**

### CODE REVIEW

File: Bandwidth.Standard/Http/Client/HttpClientConfiguration.cs [hunk 1]
```diff
@@ -131,11 +131,11 @@ public class Builder
             private TimeSpan maximumRetryWaitTime = TimeSpan.FromSeconds(0);
             private IList<int> statusCodesToRetry = new List<int>
             {
-                408, 413, 429, 500, 502, 503, 504, 521, 522, 524, 408, 413, 429, 500, 502, 503, 504, 521, 522, 524,
+                408, 413, 429, 500, 502, 503, 504, 521, 522, 524, 408, 413, 429, 500, 502, 503, 504, 521, 522, 524
             }.ToImmutableList();
             private IList<HttpMethod> requestMethodsToRetry = new List<string>
             {
-                "GET", "PUT", "GET", "PUT",
+                "GET", "PUT", "GET", "PUT"
```
Comment (ajrice6713) [inline Bandwidth.Standard/Http/Client/HttpClientConfiguration.cs:138]: Why are these duplicated? #justAPIMaticThings?

**Review by ajrice6713 — APPROVED** [review]

**Comment by ajrice6713** [comment]
@hamermike Only thing I couldnt comment on was the package version - looks like its still 1.0.0 in the .csproj file

File: Bandwidth.Standard/Http/Client/HttpClientConfiguration.cs (hunk 1 above)
Comment (hamermike) [reply Bandwidth.Standard/Http/Client/HttpClientConfiguration.cs:138]: I'm not sure why that's the case but... It's been like that and I'm scared to change it.

**Comment by hamermike** [comment]
> @hamermike Only thing I couldnt comment on was the package version - looks like its still 1.0.0 in the .csproj file

This is okay. We're using tags to version when pushing to NuGet. The version in the .csproj is ignored.

**Review by ckoegel — APPROVED** [review]

**Pull request merged**